### PR TITLE
AOA Conversion: Update Azure.MixedReality.Authentication

### PR DESCRIPTION
**Pull request by craigktreasure**
This change updates Azure.MixedReality.Authentication and prepares for release.

### CODE REVIEW

File: sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/Azure.MixedReality.ObjectAnchors.Conversion.csproj
Comment (jsquire) [inline sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/Azure.MixedReality.ObjectAnchors.Conversion.csproj:15]: Why do we need to override?  Does something prevent us from bumping the central package version?

File: sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/CHANGELOG.md
Comment (jsquire) [inline sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/CHANGELOG.md:11]: This is an unusual pattern that we don't typically see.  To me, this implies that you'll include all transitive dependency updates in your changelog going forward.   But, that also opens up the complaint that we're not tracking dependencies of dependencies.

In my opinion, it'd be better to not include this unless we have a specific need/reason for doing so.

Comment (craigktreasure) [reply sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/CHANGELOG.md:11]: Sure. I can remove them. They're required in Java land, which is why I added them.

Comment (craigktreasure) [reply sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/CHANGELOG.md:11]: Also, these aren't transitive dependencies. These are direct dependencies.

Comment (craigktreasure) [reply sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/CHANGELOG.md:11]: @jsquire What's the recommendation on direct dependencies?

File: sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/Azure.MixedReality.ObjectAnchors.Conversion.csproj
Comment (craigktreasure) [reply sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/Azure.MixedReality.ObjectAnchors.Conversion.csproj:15]: There are repo policies that require you either set the version centrally or you have to override. This library is consumed by a few other Azure SDKs and I imagine this was the solution so that each could update independently.

File: sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/CHANGELOG.md
Comment (jsquire) [reply sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/CHANGELOG.md:11]: The same; we do not typically detail version bumps unless there's some special scenario that would cause customers to have to take action because of the change.  (for example, breaking changes in a dependency)

File: sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/Azure.MixedReality.ObjectAnchors.Conversion.csproj
Comment (jsquire) [reply sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/Azure.MixedReality.ObjectAnchors.Conversion.csproj:15]: We very strongly discourage overrides outside of specific temporary needs.   To ensure that the packages in the Azure SDK work together as seamlessly as possible, we want to manage versions centrally and have packages use them.

File: sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/CHANGELOG.md
Comment (craigktreasure) [reply sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/CHANGELOG.md]: Thanks for the clarification. I've removed those notes.

**Comment by jsquire** [comment]
> @jsquire Can I get a thumbs up or thumbs down from you?

I'm not comfortable approving with the version override in place as a strategy.   If that's something that you feel strongly about, I'd like to loop in the architects and folks from EngSys for additional thoughts.

**Comment by craigktreasure** [comment]
> > @jsquire Can I get a thumbs up or thumbs down from you?
> 
> I'm not comfortable approving with the version override in place as a strategy. If that's something that you feel strongly about, I'd like to loop in the architects and folks from EngSys for additional thoughts.

That's how it works today. I'm open to the discussion, but I don't see a need for that to block this PR.

**Comment by jsquire** [comment]
> That's how it works today. I'm open to the discussion, but I don't see a need for that to block this PR.

Fair enough; I'll approve and loop in others for the discussion.

**Review by jsquire — APPROVED** [review]

**Comment by jsquire** [comment]
@KrzysztofCwalina, @tg-msft, @weshaggard:  I'd appreciate your thoughts on the approach used for package version override as a long-term strategy here.

**Pull request merged**
